common/openstack/adding-an-openstack-cloud-provider.adoc

```diff
@@ -0,0 +1,63 @@
+=========
+
+. By default, _tenant mapping_ is disabled. To enable it, set *Tenant Mapping Enabled* to *Yes*.
+
+. Select the appropriate *Zone* for the provider. By default, the zone is set to *default*.
++
+NOTE: For more information, see the definition of host aggregates and availability zones in  https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/architecture-guide/chapter-1-components#comp-compute[OpenStack Compute (nova)] in the Red Hat OpenStack Platform _Architecture Guide_.
+
+. In the *Default* tab, under *Endpoints*, configure the host and authentication details of your OpenStack provider:
+.. In *Hostname (or IPv4 or IPv6 address)*, enter the public IP or fully qualified domain name of the OpenStack Keystone service. 
++
+NOTE: The hostname required here is also the *OS_AUTH_URL* value in the *~/overcloudrc* file generated by the director (see https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/director-installation-and-usage/#sect-Accessing_the_Overcloud[Accessing the Overcloud] in Red Hat OpenStack Platform _Director Installation and Usage_ or the *~/keystonerc_admin* file generated by Packstack (see https://access.redhat.com/articles/1127153[Evaluating OpenStack: Single-Node Deployment]).
+.. In *API Port*, set the public port used by the OpenStack Keystone service. By default, OpenStack uses port 5000 for this.
+.. Select the appropriate *Security Protocol* used for authenticating with your OpenStack provider.
+.. In the *Username* field, enter the name of an OpenStack user with privileged access (for example, *admin*). Then, provide its corresponding password in the *Password* and *Confirm Password* fields.
+.. Click *Validate* to confirm {product-title} can connect to the OpenStack provider.
+
+. Next, configure how {product-title} should receive events from the OpenStack provider. Click the *Events* tab in the *Endpoints* section to start.
+* To use the Telemetry service of the OpenStack provider, select *Ceilometer*. Before you do so, the provider must first be configured accordingly. See xref:openstack-events-oc[] for details.
+* If you prefer to use the AMQP Messaging bus instead, select *AMQP*. When you do:
+ In *Hostname (or IPv4 or IPv6 address)* (of the *Events* tab, under *Endpoints*), enter the public IP or fully qualified domain name of the AMQP host.
+** In the *API Port*, set the public port used by AMQP. By default, OpenStack uses port 5672 for this.
+** In the *Username* field, enter the name of an OpenStack user with privileged access (for example, *admin*). Then, provide its corresponding password in the *Password* and *Confirm Password* fields.
+** Click *Validate* to confirm the credentials.
+
+. Click *Add* after configuring the cloud provider.
+
+[NOTE]
+====
+To collect inventory and metrics from an OpenStack environment, the {product-title} appliance requires that the adminURL endpoint for the OpenStack environment should be on a non-private network.
```

Comment:
Please check/edit: 
I think "should be" is better off worded as "must be" or "is", since the appliance requires it.